source/global.cc

```diff
@@ -23,6 +23,41 @@
 
 namespace aspect
 {
-  const double year_in_seconds  = 60*60*24*365.2425;
+  
+  const double year_in_seconds  = 60*60*24*365.2425; // s
+
+  namespace constants
+  {
+    // Constants from NIST
+    const double gas_constant = 8.3144621; // J/K/mol
+    const double avogadro = 6.02214129e23; // mol^-1
+    const double big_g = 6.67384e-11; // m^3 kg^-1 s^-2
+    
+    namespace earth
+    {
+      // Values taken from the IASP91 model
+      const double inner_core_radius = 1.2171e6; // m, 5150 km depth
+      const double core_radius = 3.482e6; // m, 2889 km depth
+      const double lower_mantle_radius = 5.711e6; // m, 660 km depth
+      const double radius = 6.371e6; // m
+      
+      // Values taken from the PREM (Dziewonski and Anderson, 1981)
+      const double g_icb_prem = 4.4002;
+      const double g_cmb_prem = 10.6823;
+      const double g_ulmb_prem = 10.0143;
+      const double g_surface_prem = 9.8156;
+      
+      // "Standard gravity", from NIST 
+      const double g = 9.80665; // average gravitational acceleration at surface ms^-2
```

Comment:
I'd rather just spell it out as `gravity`. Same for the PREM values -- and to avoid making the names too long, you can put them into a nested namespace PREM, for example.